mlflow/utils/env.py

```diff
@@ -3,8 +3,3 @@
 
 def get_env(variable_name):
     return os.environ.get(variable_name)
```

Comment:
IMO, this function is useless. We can remove this once we remove all the `get_env` calls.